### ISSUE TITLE
Feature request: support Twitter mobile 

### ISSUE THREAD

**Issue by godkarmachine** [kheina-com/Blue-Blocker#3]
Please build in support for mobile.twitter.com. I primarily use the mobile site via Firefox Nightly, and would love to use the extension to block Twitter Blue users on the go.

**Comment by Anurag-Raut** [comment]
I am not able to reproduce this issue , whenever I am trying to access mobile.twitter.com , it redirects me to twitter.com .

However I think changing twitter.com to (twitter.com | mobile.twitter.com) should work on line 1 in inject.js
 should work.
https://github.com/kheina-com/Blue-Blocker/blob/main/inject.js#L1

should I do a PR for this?

**Comment by DanielleMiu** [comment]
hey! I don't believe that is the issue, the issue is more around testing to ensure it works on the firefox mobile client. the way the regex is set up right now means that it will already match mobile urls, but I'm not sure if mobile twitter hits the same endpoints as the desktop website does
https://regexr.com/7cvme

**Comment by cooljeanius** [comment]
I have a friend who I was telling this extension about who says he's waiting on this issue to be fixed before he can use this extension

**Comment by jaredkotoff** [comment]
![Screenshot_20230510_160635_Firefox Nightly.jpg](https://github.com/kheina-com/Blue-Blocker/assets/3268576/60c48d1d-f33c-495c-866f-25640a58b6fc)

**Comment by jaredkotoff** [comment]
I tested this and it works for me.
1. Make a collection: https://addons.mozilla.org/en-US/firefox/collections/
	1. Add Blue Blocker to the collection
2. Note the ID # in the url
3. Enable extended add-on support: https://support.mozilla.org/en-US/kb/extended-add-support
4. Go to settings and add custom collection and enter the owner ID from before

**Comment by DanielleMiu** [comment]
if the add-on already works on mobile, I'll just flag it as supporting mobile next time I push a release

**Comment by jaredkotoff** [comment]
Might want to make a followup task to add a breakpoint in the styling for the popup

**Comment by cooljeanius** [comment]
@rougetimelord did you forget a closure message here?

**Comment by rougetimelord** [comment]
Whoops, seems like mobile Twitter is supported. The only thing that could be changed is adding a <br> tag to the popup if the screen width is narrow, but I'm not sure if there is a practical way to figure that out
